web/modules/custom/ai_screening_project_track/src/Helper/ProjectTrackTypeHelper.php

```diff
@@ -169,26 +169,17 @@ public function validateConfiguration(array $configuration): void {
   /**
    * Get a specific threshold.
    */
-  public function getThreshold($termId, $key, ?string $threshold = NULL): int {
+  public function getThreshold(int $termId, int $key, string $threshold): int {
```

Comment:
Passing an enum value may make this function a little easier to understand, e.g:

```suggestion
  public function getThreshold(int $termId, int $dimension, Evaluation $evaluation): int {
    $thresholds = $this->getThresholds();
    
    return $thresholds[$termId, $dimension, $evaluation->value] ?? 0;
```